ddtrace/tracer/remote_config.go

```diff
@@ -33,11 +33,12 @@ type target struct {
 }
 
 type libConfig struct {
-	Enabled       *bool             `json:"tracing_enabled,omitempty"`
-	SamplingRate  *float64          `json:"tracing_sampling_rate,omitempty"`
-	SamplingRules *[]rcSamplingRule `json:"tracing_sampling_rules,omitempty"`
-	HeaderTags    *headerTags       `json:"tracing_header_tags,omitempty"`
-	Tags          *tags             `json:"tracing_tags,omitempty"`
+	Enabled            *bool             `json:"tracing_enabled,omitempty"`
+	SamplingRate       *float64          `json:"tracing_sampling_rate,omitempty"`
+	TraceSamplingRules *[]rcSamplingRule `json:"tracing_sampling_rules,omitempty"`
+	SpanSamplingRules  *[]rcSamplingRule `json:"span_sampling_rules,omitempty"`
```

Comment:
Span sampling rules are not supported in remote config yet. If you're adding this to make it easier to support in the future (if we ever support it), that's fine, but the json field may not be called `span_sampling_rules`, since all tracing setting are prefixed with `tracing_*`.

Consider _not_ including this yet, since we don't know if we will ever add support for remote span rules or what the name for the json field would be.

Reply:
+1